### PR TITLE
Explain the difference between helm install and helm upgrade

### PR DESCRIPTION
**Pull request by JakeSCahill**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2608
Review deadline: 9 August

## Page previews

[Preview](https://deploy-preview-675--redpanda-docs-preview.netlify.app/current/manage/kubernetes/k-configure-helm-chart/#differences-between-helm-install-and-helm-upgrade)

## Checks

- [ ] New feature
- [ ] Content gap
- [x] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Review by RafalKorepta — APPROVED** [review]

File: modules/manage/pages/kubernetes/k-configure-helm-chart.adoc [hunk 1]
```diff
@@ -235,6 +235,29 @@ helm show values redpanda/console
 
 For default values and documentation for configuration options, see the https://artifacthub.io/packages/helm/redpanda-data/console?modal=values[`values.yaml`^] file.
 
+== Differences between helm install and helm upgrade
+
+When managing Redpanda deployments with Helm, it's important to understand the differences between `helm install` and `helm upgrade`, particularly in how they handle cluster configuration overrides.
+
+=== Reinstall Redpanda with `helm install`
+
+When reinstalling Redpanda with `helm install`, cluster configuration overrides specified in the Helm values may not take effect due to PersistentVolumeClaim (PVC) retention. By default, most PVCs are retained when a Helm release is uninstalled. As a result, when Redpanda is re-installed, the previously created PVCs are adopted, restoring the state of the previous cluster. This adoption results in the new `bootstrap.yaml` file being ignored and the `post_upgrade` job not running. The `post_upgrade` job is a component in the Helm chart that ensures the application of configuration overrides during an upgrade.
```
Comment (JakeSCahill) [inline modules/manage/pages/kubernetes/k-configure-helm-chart.adoc]:
```suggestion
When reinstalling Redpanda with `helm install`, cluster configuration overrides specified in the Helm values may not take effect due to PersistentVolumeClaim (PVC) retention. By default, most PVCs are retained when a Helm release is uninstalled. As a result, when Redpanda is reinstalled, the previously created PVCs are adopted, restoring the state of the previous cluster. This adoption results in the new `bootstrap.yaml` file being ignored and the `post_upgrade` job not running. The `post_upgrade` job is a component in the Helm chart that ensures the application of configuration overrides during an upgrade.
```

File: modules/manage/pages/kubernetes/k-configure-helm-chart.adoc [hunk 2]
```diff
@@ -235,6 +235,29 @@ helm show values redpanda/console
 
 For default values and documentation for configuration options, see the https://artifacthub.io/packages/helm/redpanda-data/console?modal=values[`values.yaml`^] file.
 
+== Differences between helm install and helm upgrade
+
+When managing Redpanda deployments with Helm, it's important to understand the differences between `helm install` and `helm upgrade`, particularly in how they handle cluster configuration overrides.
+
+=== Reinstall Redpanda with `helm install`
```
Comment (JakeSCahill) [inline modules/manage/pages/kubernetes/k-configure-helm-chart.adoc]:
```suggestion
=== Reinstall Redpanda
```

File: modules/manage/pages/kubernetes/k-configure-helm-chart.adoc [hunk 3]
```diff
@@ -235,6 +235,29 @@ helm show values redpanda/console
...
+. Delete the existing PVCs before re-installing Redpanda:
++
+[,bash]
+----
+kubectl delete pvc -l app=redpanda --namespace <namespace>
+----
+
+. Execute the `helm install` command to reinstall Redpanda with a clean state.
+
+=== Configure an existing cluster with `helm upgrade`
```
Comment (JakeSCahill) [inline modules/manage/pages/kubernetes/k-configure-helm-chart.adoc]:
```suggestion
=== Configure an existing cluster
```

File: modules/manage/pages/kubernetes/k-configure-helm-chart.adoc [hunk 4]
```diff
@@ -235,6 +235,29 @@ helm show values redpanda/console
 
 For default values and documentation for configuration options, see the https://artifacthub.io/packages/helm/redpanda-data/console?modal=values[`values.yaml`^] file.
 
+== Differences between helm install and helm upgrade
+
+When managing Redpanda deployments with Helm, it's important to understand the differences between `helm install` and `helm upgrade`, particularly in how they handle cluster configuration overrides.
```
Comment (JakeSCahill) [inline modules/manage/pages/kubernetes/k-configure-helm-chart.adoc:240]:
```suggestion
When managing Redpanda deployments with Helm, it's important to understand the differences between `helm install` and `helm upgrade`, particularly in how they handle cluster configuration overrides.

Use `helm install` to install or reinstall Redpanda. Use `helm upgrade` to reconfigure an existing deployment.
```

File: modules/manage/pages/kubernetes/k-configure-helm-chart.adoc [hunk 5]
```diff
@@ -235,6 +235,31 @@ helm show values redpanda/console
...
+
+Use `helm install` to install or reinstall Redpanda. Use `helm upgrade` to reconfigure an existing deployment.
+
+=== Reinstall Redpanda
+
+When reinstalling Redpanda with `helm install`, cluster configuration overrides specified in the Helm values may not take effect due to PersistentVolumeClaim (PVC) retention. By default, most PVCs are retained when a Helm release is uninstalled. As a result, when Redpanda is reinstalled, the previously created PVCs are adopted, restoring the state of the previous cluster. This adoption results in the new `bootstrap.yaml` file being ignored and the `post_upgrade` job not running. The `post_upgrade` job is a component in the Helm chart that ensures the application of configuration overrides during an upgrade.
```
Comment (asimms41) [inline modules/manage/pages/kubernetes/k-configure-helm-chart.adoc]:
```suggestion
When reinstalling Redpanda with `helm install`, cluster configuration overrides specified in the Helm values may not take effect due to PersistentVolumeClaim (PVC) retention. 

By default, most PVCs are retained when a Helm release is uninstalled. As a result, when Redpanda is reinstalled, the previously created PVCs are adopted, restoring the state of the previous cluster. This adoption results in the new `bootstrap.yaml` file being ignored and the `post_upgrade` job not running. The `post_upgrade` job is a component in the Helm chart that applies configuration overrides during an upgrade.
```

File: modules/manage/pages/kubernetes/k-configure-helm-chart.adoc [hunk 6]
```diff
@@ -235,6 +235,31 @@ helm show values redpanda/console
...
+[,bash]
+----
+kubectl delete pvc -l app=redpanda --namespace <namespace>
+----
+
+. Execute the `helm install` command to reinstall Redpanda with a clean state.
+
+=== Configure an existing cluster
+
+During a `helm upgrade`, the `post_upgrade` job is triggered, ensuring that the latest overrides are applied to the cluster.
```
Comment (asimms41) [inline modules/manage/pages/kubernetes/k-configure-helm-chart.adoc]:
```suggestion
During a `helm upgrade`, the `post_upgrade` job is triggered, which applies the latest overrides to the cluster.
```

File: modules/manage/pages/kubernetes/k-configure-helm-chart.adoc [hunk 7]
```diff
@@ -235,6 +235,31 @@ helm show values redpanda/console
...
+
+Use `helm install` to install or reinstall Redpanda. Use `helm upgrade` to reconfigure an existing deployment.
+
+=== Reinstall Redpanda
+
+When reinstalling Redpanda with `helm install`, cluster configuration overrides specified in the Helm values may not take effect due to PersistentVolumeClaim (PVC) retention. By default, most PVCs are retained when a Helm release is uninstalled. As a result, when Redpanda is reinstalled, the previously created PVCs are adopted, restoring the state of the previous cluster. This adoption results in the new `bootstrap.yaml` file being ignored and the `post_upgrade` job not running. The `post_upgrade` job is a component in the Helm chart that ensures the application of configuration overrides during an upgrade.
+
+To ensure the new installation does not adopt the old PVCs and restore the previous state:
+
+. Delete the existing PVCs before re-installing Redpanda:
```
Comment (asimms41) [inline modules/manage/pages/kubernetes/k-configure-helm-chart.adoc]:
```suggestion
. Delete the existing PVCs before reinstalling Redpanda:
```

**Review by asimms41 — CHANGES_REQUESTED** [review]
Looks great! Just a couple of suggestions.

**Review by chrisseto — APPROVED** [review]
Beautifully crafted 🎉 Thanks Jake!

**Review by asimms41 — APPROVED** [review]
Looks great! Thanks for the updates.

**Pull request merged**